HELP I accidently revoked my DIM from Bungie and cant seem to use it

**What version of DIM are you using?**
Find the version in "About DIM" from the menu (the button with three bars on the upper left of the screen). Is the DIM icon orange, or blue? Orange 

**What Browser and OS are you using?**
It helps to know what type of browser and what version you're using. Have you installed DIM as an app through Chrome or Safari? Chrome 

**Describe the bug and how to make it happen**
Tell us what's wrong. A great format for bug reports is "I did X, and I expected Y, but instead Z happened." If it helps illustrate the issue, please add screenshots to help explain your problem.
 I accidently revoked my DIM from bungie and I cant seem to reactivate it
**Include devtools console logs**
If you can, open the devtools console (ctrl+shift+J on PC and cmd+option+J on Mac) and paste what you see in the window that pops up:

```
Paste console logs here
```

2www.bungie.net/platform/app/oauth/token/:1 Failed to load resource: the server responded with a status of 400 ()
d2-stores.ts:263 Error loading stores Error: Unknown error getting response token: server_error, AuthorizationRecordRevoked
    at authenticated-fetch.ts:158
    at async l (authenticated-fetch.ts:33)
    at async f (bungie-service-helper.ts:67)
    at async p (destiny2-api.ts:109)
    at async Promise.all (index 3)
    at async https:/app.destinyitemmanager.com/main-23c81a.js?__WB_REVISION__=577497879ffde4cf0f5ccfedaf59d9ab:1
(anonymous) @ d2-stores.ts:263
inventory:1 A cookie associated with a cross-site resource at http://cdn.syndication.twimg.com/ was set without the `SameSite` attribute. A future release of Chrome will only deliver cookies with cross-site requests if they are set with `SameSite=None` and `Secure`. You can review cookies in developer tools under Application>Storage>Cookies and see more details at https://www.chromestatus.com/feature/5088147346030592 and https://www.chromestatus.com/feature/5633521622188032.
inventory:1 A cookie associated with a cross-site resource at https://twitter.com/ was set without the `SameSite` attribute. A future release of Chrome will only deliver cookies with cross-site requests if they are set with `SameSite=None` and `Secure`. You can review cookies in developer tools under Application>Storage>Cookies and see more details at https://www.chromestatus.com/feature/5088147346030592 and https://www.chromestatus.com/feature/5633521622188032.
inventory:1 A cookie associated with a cross-site resource at http://twitter.com/ was set without the `SameSite` attribute. A future release of Chrome will only deliver cookies with cross-site requests if they are set with `SameSite=None` and `Secure`. You can review cookies in developer tools under Application>Storage>Cookies and see more details at https://www.chromestatus.com/feature/5088147346030592 and https://www.chromestatus.com/feature/5633521622188032.
inventory:1 A cookie associated with a cross-site resource at http://syndication.twitter.com/ was set without the `SameSite` attribute. A future release of Chrome will only deliver cookies with cross-site requests if they are set with `SameSite=None` and `Secure`. You can review cookies in developer tools under Application>Storage>Cookies and see more details at https://www.chromestatus.com/feature/5088147346030592 and https://www.chromestatus.com/feature/5633521622188032.
www.bungie.net/platform/app/oauth/token/:1 Failed to load resource: the server responded with a status of 400 ()
authenticated-fetch.ts:158 Uncaught (in promise) Error: Unknown error getting response token: server_error, AuthorizationRecordRevoked
    at authenticated-fetch.ts:158
    at async [[https:/app.destinyitemmanager.com/main-23c81a.js?__WB_REVISION__=577497879ffde4cf0f5ccfedaf59d9ab:1[](url)]](url
![image](https://user-images.githubusercontent.com/68758074/88438081-c7c2ac00-cdff-11ea-9fa4-e0115b3364ec.png)
)

--- a/src/app/bungie-api/authenticated-fetch.ts
+++ b/src/app/bungie-api/authenticated-fetch.ts
@@ -77,7 +77,11 @@ async function responseIndicatesBadToken(response: Response) {
     (data.ErrorCode === PlatformErrorCodes.AccessTokenHasExpired ||
       data.ErrorCode === PlatformErrorCodes.WebAuthRequired ||
       // (also means the access token has expired)
-      data.ErrorCode === PlatformErrorCodes.WebAuthModuleAsyncFailed)
+      data.ErrorCode === PlatformErrorCodes.WebAuthModuleAsyncFailed ||
+      data.ErrorCode === PlatformErrorCodes.AuthorizationRecordRevoked ||
+      data.ErrorCode === PlatformErrorCodes.AuthorizationRecordExpired ||
+      data.ErrorCode === PlatformErrorCodes.AuthorizationCodeStale ||
+      data.ErrorCode === PlatformErrorCodes.AuthorizationCodeInvalid)
   );
 }
 

